src/Security/samples/PathSchemeSelection/PathSchemeSelection.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IsTestAssetProject>true</IsTestAssetProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore" />
+    <Reference Include="Microsoft.AspNetCore.Authentication.Cookies" />
+    <Reference Include="Microsoft.AspNetCore.Mvc" />
+    <Reference Include="Microsoft.AspNetCore.StaticFiles" />
+    <Reference Include="Microsoft.NET.Sdk.Razor" PrivateAssets="All" />
```

Comment:
@natemcmaster these references were required in netcoreapp3.0. I remember you saying Microsoft.NET.Sdk.Web would bring it in.

Reply:
This is fine for now. We'll clean this up later. We need to cleanup a bunch of other projects in master, and we can do that in one sweep.